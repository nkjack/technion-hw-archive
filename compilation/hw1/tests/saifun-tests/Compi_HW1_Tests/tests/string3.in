""
"it's a cold cold night"
"G00D\0H N0!"
"GOOD\OH NO"
