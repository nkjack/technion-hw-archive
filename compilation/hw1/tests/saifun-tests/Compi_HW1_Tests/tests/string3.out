1 STRING 
2 STRING it's a cold cold night
3 STRING G00D\0H N0!
Error undefined escape sequence O
